recipes/parsplice/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "parsplice" %}
+{% set version = "1.1.1" %}
```

Comment:
Can you change the version to be 1.1.2.dev0 ?

Reply:
Done